src/track/cue.cpp

```diff
@@ -64,7 +72,7 @@ Cue::Cue(TrackId trackId, mixxx::AudioSignal::SampleRate sampleRate, const mixxx
           m_sampleEndPosition(Cue::kNoPosition),
           m_iHotCue(Cue::kNoHotCue),
           m_label(cueInfo.getLabel()),
-          m_color(Color::kPredefinedColorsSet.predefinedColorFromRgbColor(cueInfo.getColor())) {
+          m_color(cueInfo.getColor().value_or(kDefaultCueColor)) {
```

Comment:
is value_or() supported by our ancient C++ compilers and libs?

Reply:
The CI builds passed, so I guess that this is the case.